src/components/MetadataSources/MetadataSourceView.js

```diff
@@ -16,14 +16,22 @@ import {
   Row,
 } from '@folio/stripes/components';
 import {
-  IfPermission
+  IfPermission,
 } from '@folio/stripes/core';
 
 import SourceInfoView from './SourceInfo/SourceInfoView';
 import SourceManagementView from './SourceManagement/SourceManagementView';
 import SourceTechnicalView from './SourceTechnical/SourceTechnicalView';
 
 class MetadataSourceView extends React.Component {
+  static manifest = Object.freeze({
```

Comment:
Please remove commented-out code.
In addition, the manifest seems to be unused, thus you can delete it.

Reply:
manifest has been removed